### ISSUE TITLE
Could not read file error after upgrading to 1.10.0

### ISSUE THREAD

**Issue by bigardone** [kipcole9/money_sql#37]
Hi @kipcole9 👋🏼 
After upgrading to `1.10.0`, I get the following error while trying to run the migration for `min` and `max`:

```
11:24:35.472 [info] == Running 20231102101154 Marko.Repo.Migrations.AddPostgresMoneyMinmaxFunctions.up/0 forward
** (File.Error) could not read file "priv/SQL/postgres/get_currency_code_type.sql": no such file or directory
    (elixir 1.15.5) lib/file.ex:358: File.read!/1
    (ex_money_sql 1.10.0) lib/money/migration.ex:19: Money.Migration.postgres_money_with_currency_type/1
    (ex_money_sql 1.10.0) lib/money/migration.ex:5: Money.Migration.adjust_for_type/2
    priv/repo/migrations/20231102101154_add_postgres_money_minmax_functions.exs:42: Marko.Repo.Migrations.AddPostgresMoneyMinmaxFunctions.up/0
    (ecto_sql 3.10.2) lib/ecto/migration/runner.ex:289: Ecto.Migration.Runner.perform_operation/3
    (stdlib 5.0.2) timer.erl:270: :timer.tc/2
    (ecto_sql 3.10.2) lib/ecto/migration/runner.ex:25: Ecto.Migration.Runner.run/8
    (ecto_sql 3.10.2) lib/ecto/migrator.ex:366: Ecto.Migrator.attempt/8
```

I saw [a similar](https://github.com/kipcole9/money_sql/issues/1) closed issue from some years ago. I think the issue is here:

https://github.com/kipcole9/money_sql/blob/493de0770d3ac56cef2423ed7f0ff2e3d019178e/lib/money/migration.ex#L19

Maybe it should be using something like https://github.com/kipcole9/money_sql/blob/493de0770d3ac56cef2423ed7f0ff2e3d019178e/lib/money/ddl.ex#L256 instead.

**Comment by kipcole9** [comment]
Thanks for the issue. Yes, thats not good at all!  Will fix in the next few hours.

**Comment by bigardone** [comment]
I can create a PR if you want :)

**Comment by kipcole9** [comment]
Always gratefully accepted!Sent from my iPhoneOn 2 Nov 2023, at 18:43, Ricardo García Vega ***@***.***> wrote:﻿
I can create a PR if you want :)

**Comment by kipcole9** [comment]
I've published [ex_money_sql version 1.10.1](https://hex.pm/packages/ex_money_sql/1.10/1) with the following changelog entry:

### Bug Fixes

* Fix compilation warnings on Elixir 1.16.

* Fix migration generator for `money_with_currency` type. Thanks to @bigardone for the issue and PR. Closes #37, closes #38.

Thanks again for the support and the PR.
